data/Hints/league.json

```
@@ -43,17 +58,17 @@
         "goal":             {"order": 4, "weight": 0.0, "fixed":   5, "copies": 2},
         "barren":           {"order": 5, "weight": 0.0, "fixed":   3, "copies": 2},
         "entrance":         {"order": 6, "weight": 0.0, "fixed":   4, "copies": 2},
-        "sometimes":        {"order": 7, "weight": 0.0, "fixed": 100, "copies": 2},
-        "random":           {"order": 8, "weight": 9.0, "fixed":   0, "copies": 2},
+        "dual":             {"order": 7, "weight": 1.5, "fixed": 0, "copies": 2},
```

Let's keep the indentation consistent.

```suggestion
        "dual":             {"order": 7, "weight": 1.5, "fixed":   0, "copies": 2},
```